craco.config.cjs

```diff
@@ -44,8 +46,13 @@ module.exports = {
     pluginOptions(eslintConfig) {
       return Object.assign(eslintConfig, {
         cache: true,
-        cacheLocation: 'node_modules/.cache/eslint/',
+        cacheLocation: 'node_modules/.cache/.eslintcache',
         ignorePath: '.gitignore',
+        // Clear the create-react-app overrides to use the defaults.
+        formatter: undefined,
```

Comment:
nit: why was this not required before?

Reply:
I've updated the comment to be more explicit. CRA overrides these properties to use the eslint that ships with react-scripts, but doing so invalidates our lint cache.